### ISSUE TITLE
QWT in debug mode

### ISSUE THREAD

**Issue by masschel** [kenfred/conan-qwt#8]
First of all, thanks for creating this conan file. And even better, it is now part of the Conan.io repo :-).

There are two changes that are required to make it work in Debug mode for Visual Studio:

    def package(self):
        self.copy("FindQwt.cmake", ".", ".")
        self.copy("*.h", dst="include", src=os.path.join("qwt-%s" % self.version, "src"))

       // Add an asterix to include the qwtd.lib file

        self.copy("*qwt*.lib", dst="lib", keep_path=False)
        self.copy("*.dll", dst="bin", keep_path=False)
        self.copy("*.so*", dst="lib", keep_path=False, symlinks=True)
        self.copy("*.dylib", dst="lib", keep_path=False)
        self.copy("*.a", dst="lib", keep_path=False)

    def package_info(self):
        self.cpp_info.libs = ["qwt"]

        // Add the following lines to use the debug qwtd file.

        if self.settings.os == "Windows" and self.settings.build_type == "Debug" and self.settings.compiler == "Visual Studio":
            self.cpp_info.libs = [lib + "d" for lib in self.cpp_info.libs]

Like this, it works like a charm for me :-). Or am I doing sth wrong, and the vanilla one should work out-of-the-box?

**Comment by kenfred** [comment]
You are correct. I have some potential fixes for this, but would suggest the conan center index may be a better bet for you. Is there a reason to avoid the "official" one in conan center?

**Comment by masschel** [comment]
Ik have no reason at all not to use the conan center one, and would even prefer it, it fits better the normal flow of our development. Qwt is only recently added to Conan, and by the time we fetched it, the file was the same as on your repo, so I thought it was fetched from there. I re-pulled the Conan packages, and saw there were changes that addressed this issue, so indeed, me happy!

Thanks for providing this Conan installer! It helped our team a lot 😊.
